src/app/profile/profile.tpl.html

```diff
@@ -46,7 +46,7 @@ <h4 class="panel-title  border-bottom-small visible">Your Name</h4>
                               <input name="nameMiddleInitial" type="text" class="form-control  form-control-subtle middle" disabled ng-model="$ctrl.donorDetails.name['middle-initial']">
                               <input name="nameFamilyName" type="text" class="form-control  form-control-subtle last" disabled ng-model="$ctrl.donorDetails.name['family-name']">
 
-                              <select name="nameSuffix" class="form-control form-control-subtle suffix" ng-model="$ctrl.donorDetails.name.suffix">
+                              <select name="nameSuffix" class="form-control form-control-subtle suffix" disabled ng-model="$ctrl.donorDetails.name.suffix">
```

Comment:
The others aren't... I think the intention is that they will never be edited by our app. But they are all still wired up with ng-model and I think sent to the api. I guess this code could be used if that requirement changes in the future but right now I guess the code isn't needed. I'm not sure what requirements Denys got when he initially implemented this.